gradle.properties

```diff
@@ -17,7 +17,3 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
-# Enables namespacing of each library's R class so that its R class includes only the
-# resources declared in the library itself and none from the library's dependencies,
-# thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
```

Comment:
Good, android.nonTransitiveRClass will now default to false in AGP 8.0.0. Did you remove it manually or use the AGP Upgrade Assistant?

Reply:
Manually.

Reply:
Then you know enough to consider these details.